tensorflow/g3doc/get_started/os_setup.md

```diff
@@ -1112,6 +1112,17 @@ Step #1 might already solve the problem, however if it still persists, execute s
 This issue occurs with new Anaconda installations when `pip` tries to remove `easy-install.pth`. 
 This file is not included in Anaconda packages, which causes the `pip` installation to fail.
 
+#### Cupti_wrapper.cc: Could not find cuptiActivityRegisterCallbacksin libcupti DSO
+
+If running TensorFlow Python script you encounter the error:
+```
+c:\tf_jenkins\home\workspace\nightly-win\device\gpu\os\windows\tensorflow\core\platform\default\gpu\cupti_wrapper.cc:59] Check failed: ::tensorflow::Status::OK() == (::tensor     flow::Env::Default()->GetSymbolFromLibrary( GetDsoHandle(), kName, &f  )) (OK vs. Not found: cuptiActivityRegisterCallbacks not found)could not find cuptiActivityRegisterCallb     acksin libcupti DSO
+```
+
+Add `<path-to-cuda-folder>\NVIDIA GPU Computing Toolkit\CUDA\v8.0\extras\CUPTI\libx64` to your `PATH`.
+
+This issue occurs because on CUDA 8.0 the location of the files `cupti64_80.dll` and `cupti.lib` is not on `PATH` by default.
```

Comment:
Nit: I think it's only `cupti64_80.dll` that's relevant here. (The `.lib` is just used at build-time.)